content/v1.13/concepts/patch-and-transform.md

```diff
@@ -1571,10 +1572,11 @@ patches:
 
 #### Trim suffix
 
-The {{<hover label="typeRegex" line="8">}}type: TrimSuffix{{</hover>}} removes
-the matching string and all proceeding characters. 
+The {{<hover label="typeTrimS" line="8">}}type: TrimSuffix{{</hover>}} uses 
+Go's [TrimSuffix](https://pkg.go.dev/strings#TrimPrefix) and removes characters 
```

Comment:
this should link to the TrimSuffix function - i think this problem exists for all versions